app/views/works/_edit_javascript.html.erb

```diff
@@ -0,0 +1,151 @@
+<script>
+  $(function() {
+    var isOrcid = function(value) {
+      const regex = new RegExp(/^\d\d\d\d-\d\d\d\d-\d\d\d\d-\d\d\d\d$/);
+      return regex.test(value);
+    };
+
+    var incrementCounter = function(elementId) {
+      var counter = parseInt($(elementId)[0].value, 10);
+      counter++
+      $(elementId)[0].value = counter
+      return counter
+    }
+
+    var addCreatorHtml = function(num, orcid, givenName, familyName) {
+      var rowId = `creator_row_${num}`;
+      var orcidId = `orcid_${num}`;
+      var givenNameId = `given_name_${num}`;
+      var familyNameId = `family_name_${num}`;
+      var rowHtml = `<tr id="${rowId}">
+        <td>
```

Comment:
@hectorcorrea I was wondering if you could do something like `rowHtml = '<%= render partial %>' in the javascript.  Since it is ERB I think it would work.  I utilized a similar technique in the ORCID PR you and I did together (https://github.com/pulibrary/pdc_describe/blob/main/app/views/works/_form.html.erb#L186).

Reply:
That approach will work for rendering the existing creators on the record, but I am not sure it will work for records added on the fly by the user. But we can certainly try that approach as a separate PR to keep this one from growing.